Source/WebCore/page/Quirks.cpp

```diff
@@ -91,7 +91,7 @@ static inline OptionSet<AutoplayQuirk> allowedAutoplayQuirks(Document& document)
 {
     auto* loader = document.loader();
     if (!loader)
-        return { };
+        return {};
```

Comment:
I think the typical WebKit style is to leave a space.

Reply:
More specifically al these changes are not part of the initial code, or necessary for this pull request. 
@filippofinke would it be possible to just update this PR one more time to not change the `{ }`. Thanks!